src/commands/sourcemaps/inject.rs

```diff
@@ -35,22 +34,25 @@ pub fn make_command(command: Command) -> Command {
 }
 
 pub fn execute(matches: &ArgMatches) -> Result<()> {
-    let path = matches.get_one::<OsString>("path").unwrap();
+    let path = matches.get_one::<String>("path").unwrap();
 
     let mut collected_paths = Vec::new();
     for entry in WalkDir::new(path) {
-        let entry = entry.unwrap();
+        let entry = match entry {
+            Ok(entry) => entry,
+            Err(e) => {
+                error!("{e}");
```

Comment:
```suggestion
                debug!("skipping file {}", path.display());
                debug!("error: {}", e);
```

Reply:
Not sure if there's a way to print something better than `entry`. This is inline with what we do with difs. 

https://github.com/getsentry/sentry-cli/blob/f8dfc402ab4711a421160e5fc74e39b0c4d17ed2/src/utils/dif_upload.rs#L561-L583